docs/technical_specs/core/util/tags.rst

```diff
@@ -0,0 +1,109 @@
+Tagging Classes
+===============
+
+IDAES contains classes for tagging model quantities and grouping them.  The tags
+allow a convenient short cut to important model inputs and outputs.  There are
```

Comment:
Minor comment, but I would combine the last two statements into a full sentence for better flow.